networking/amplify/backend/api/mobile/schema.graphql

```diff
@@ -9,28 +9,20 @@ type Mutation {
   )
 }
 
-input CreateGameSessionFromTemplateInput {
-  gameTemplateId: ID!
-}
-
-# define a GameTemplate in amplify and graphql to list all gametemplates sorted by updatedAt
-type GameTemplate @model
+type PublicGameTemplate @model
 @auth(
   rules: [
     { allow: public, operations: [read], provider: iam  },
-    { allow: private, operations: [read, create, update, delete], provider: userPools }
+    { allow: owner, provider: userPools },
+    { allow: private, operations: [read], provider: userPools },  
   ]
 )
 @aws_iam
 @aws_cognito_user_pools
 {
   id: ID!
+  owner: String
```

Comment:
important to add owner in here as an optional parameter. On creation, amplify populates this field with the owner information, but it needs to be specified as a field that can be null upon initial creation. Ugh